docs/build_and_test.md

```diff
@@ -1,32 +1,23 @@
 ## 1.Clone RDASApp
-In the directory that you want to install RDASApp:  
 ```
 git clone --recurse-submodules https://github.com/NOAA-EMC/RDASApp.git
```

Comment:
I wonder if we should add this line here? If not just mention this command and suggest it *could* be used here.
```
git submodule update --init --remote
```
This will update all the submodules to the most recent instead of using the versions that haven't been updated in awhile. Although, I think we will start updating submodules more frequently in the future. Otherwise it looks good to me.

Reply:
@delippi Thanks for your review! 
As discussed in the issue https://github.com/NOAA-EMC/RDASApp/issues/65 and agreed in a small group tag up, we will use snapshots of submodules (instead of moving head commits) for each RDASApp commit. If a submodule needs a new version, a PR should be created in RDASApp to update it.  @ShunLiu-NOAA and @hu5970 have been working on a draft for the code management strategy and will share it with us soon.